server/application-server/src/main/java/de/tum/in/www1/hephaestus/chat/session/Session.java

```diff
@@ -0,0 +1,38 @@
+package de.tum.in.www1.hephaestus.chat.session;
+
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import org.springframework.lang.NonNull;
+import jakarta.persistence.*;
+import de.tum.in.www1.hephaestus.chat.message.Message;
+import de.tum.in.www1.hephaestus.gitprovider.user.User;
+
+@Entity
+@Table(name = "session")
+@Getter
+@Setter
+@ToString(callSuper = true)
+@NoArgsConstructor
+public class Session {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @OrderColumn(name = "message_order")
```

Comment:
I'm probably missing something here, but `Message` doesn't have such a column?
Probably ordering after `sentAt` makes the most sense here?